pwa-devdocs/src/index.html

```diff
@@ -116,7 +116,7 @@ <h3>
       <div class="column">
         <article class="repository tile">
           <h3>
-            <a href="https://github.com/magento-research/pwa-studio/tree/release/2.0/packages/pwa-buildpack">Buildpack source</a>
+            <a href="https://github.com/magento-research/pwa-studio/tree/develop/packages/pwa-buildpack">Buildpack source</a>
```

Comment:
You may or may not want to point these links to `master`, which will have the latest released version of these source files.